Formula/yarn.rb

```diff
@@ -24,6 +22,7 @@ def install
     (bin/"yarnpkg").write_env_script "#{libexec}/bin/yarn.js",
       PREFIX:            HOMEBREW_PREFIX,
       NPM_CONFIG_PYTHON: "/usr/bin/python"
+    inreplace "#{libexec}/lib/cli.js", "/usr/lib", HOMEBREW_PREFIX
```

Comment:
Should this be
```suggestion
    inreplace "#{libexec}/lib/cli.js", "/usr/local", HOMEBREW_PREFIX
```
? `brew` doesn't replace `/usr/lib`. Squash your commits together with the first when you make this change please. Thanks!

Reply:
oof thanks for the catch, i was just on autopilot when i typed that